pypit/arutils.py

```diff
@@ -400,6 +400,27 @@ def func_val(c, x, func, minv=None, maxv=None):
                    "Please choose from 'polynomial', 'legendre', 'chebyshev', 'bspline'")
 
 
+def calc_fit_rms(xfit, yfit, fit, func, fmin=None, fmax=None):
+    """ Simple RMS calculation
+    Parameters
+    ----------
+    xfit : ndarray
+    yfit : ndarray
+    fit : coefficients
+    func : str
+    fmin : float, optional
+    fmax : float, optional
+
+    Returns
+    -------
+    rms : float
+
+    """
+    values  = func_val(fit, xfit, func, minv=fmin, maxv=fmax)
+    rms = np.sqrt(np.sum((yfit-values)**2)/len(xfit))
```

Comment:
How about rms = np.std(yfit-values) ? 

From the docs;
The standard deviation is the square root of the average of the squared deviations from the mean, i.e., std = sqrt(mean(abs(x - x.mean())**2)).